source/AI.cpp

```diff
@@ -3838,7 +3893,7 @@ void AI::IssueOrders(const PlayerInfo &player, const Orders &newOrders, const st
 		for(const Ship *ship : ships)
 		{
 			// Never issue orders to a ship to target itself.
-			if(ship == newTarget)
+			if(targetShip && ship == targetShip)
```

Comment:
I don't think you need to check if targetShip here

Reply:
I think you're right; this was a remnant from when my game was crashing and I wasn't sure why.  I now have a proper debugger installed.  I will fix this.